pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -253,9 +253,12 @@ func (cgc *containerGC) evictContainers(gcPolicy kubecontainer.ContainerGCPolicy
 	return nil
 }
 
-// evictSandboxes evicts all sandboxes that are evictable. Evictable sandboxes are:
-// containing no containers at all and not the latest sandbox if it is belonging to
-// an existing pod.
+// evictSandboxes remove all evictable sandboes. An evictable sandbox must
```

Comment:
I believe this function is wrong.

For container garbage collection, what we are doing:
1) get `evictableContainers` (prune out running ones);
2) remove all or keep latest `evictableContainers` based on whether the pod is deleted.

I quite don't understand what the PR is doing now.